dot/sync/bootstrap_syncer_test.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package sync
+
+import (
+	"testing"
+
+	"github.com/ChainSafe/gossamer/dot/types"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_bootstrapSyncer_handleWorkerResult(t *testing.T) {
+	t.Parallel()
+
+	blockStateBuilderEmpty := func(ctrl *gomock.Controller) BlockState {
+		return NewMockBlockState(ctrl)
+	}
```

Comment:
nit you could use `NewMockBlockState` directly

Reply:
When I try to use `NewMockBlockState` directly I get this:
```
cannot use NewMockBlockState (value of type func(ctrl *gomock.Controller) *MockBlockState) as type func(ctrl *gomock.Controller) BlockState in struct literal
```
however I did move the function to inline with test structs.

Reply:
My bad! Indeed, this (oddly) doesn't work.